mysql-test/suite/rpl/t/rpl_rewrite_db_sys_vars.test

```diff
@@ -82,10 +85,11 @@ select * from test_replica.my_table;
 --let $diff_tables=master:test_master.my_table,slave:test_replica.my_table
 --source include/diff_tables.inc
 
---echo # Dynamic updates to the replication filter should be lost after server restart (but they are not ?)
+--echo # Update of values on primary for DB not set in replication_rewrite_db
 connection master;
 use x;
 insert into my_table values (314);
+select * from my_table;
```

Comment:
I think it would be good to also show the contents where this is found to be replicated on the slave. I.e., `x.my_table` should have `314` in it.

Reply:
Plan is to show that below together with value 1000.